src/interface/home/ReportSelectionHeader.js

```diff
@@ -21,25 +27,54 @@ class ReportSelectionHeader extends React.PureComponent {
   constructor(props) {
     super(props);
     this.state = {
-      reportSelectionActive: true,
+      searchType: STATE_SEARCH_REPORT,
     };
-    this.handleToggleCharacterSearchClick = this.handleToggleCharacterSearchClick.bind(this);
-    this.handleToggleReportSelectorClick = this.handleToggleReportSelectorClick.bind(this);
+    this.handleCharacterSearchClick = this.handleCharacterSearchClick.bind(this);
+    this.handleReportSearchClick = this.handleReportSearchClick.bind(this);
+    this.handleGuildSearchClick = this.handleGuildSearchClick.bind(this);
   }
 
-  handleToggleCharacterSearchClick(e) {
+  handleCharacterSearchClick(e) {
     e.preventDefault();
     this.setState({
-      reportSelectionActive: false,
+      searchType: STATE_SEARCH_CHAR,
     });
   }
-  handleToggleReportSelectorClick(e) {
+
+  handleReportSearchClick(e) {
     e.preventDefault();
     this.setState({
-      reportSelectionActive: true,
+      searchType: STATE_SEARCH_REPORT,
     });
   }
 
+  handleGuildSearchClick(e) {
+    e.preventDefault();
+    this.setState({
+      searchType: STATE_SEARCH_GUILD,
+    });
+  }
+
+  renderSearch() {
+    switch (this.state.searchType) {
+      case STATE_SEARCH_CHAR:
+        return (<>
+          <NameSearch type={SearchType.CHARACTER} />
+          <br />
+          <Warning>
+            <Trans>
+              The character page will only show fights that have been ranked by Warcraft Logs. Wipes are not included and during busy periods there might be a delay before new reports appear. You can still analyze these fights by manually finding the report on Warcraft Logs and using the report link.
+            </Trans>
+          </Warning>
```

Comment:
Do guilds have the same delay limitation? If so, we might need a warning

Reply:
No idea... but I personally think the warning for characters probably shouldnt be on the front page as well. All of the relevant information is in the guild/character page error messages...